app/controllers/users_controller.rb

```diff
@@ -1,4 +1,11 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: [:index, :edit, :update, :destroy]
```

Comment:
```suggestion
  before_action :logged_in_user, only: %i[index edit update destroy]
```

シンボルや文字列の配列はこのように%記法を用いるのが一般的です。